### PR TITLE
crude suspend in Android

### PR DESCRIPTION
**Pull request by elmerohueso**
Send virtual Power button keypress to suspend Android (the best/only method I've found).  This has worked for me using Python scripts in XBMC, so it should also here.

### CODE REVIEW

**Comment by koying** [comment]
Good idea. No more crude than the others really ;) 
I'll give it a try.

File: xbmc/powermanagement/android/AndroidPowerSyscall.cpp [hunk 1]
```diff
@@ -29,6 +29,8 @@ CAndroidPowerSyscall::CAndroidPowerSyscall()
 {
   m_isRooted = false;
   m_su_path = "/system/bin/su";
+  m_hasCEC = false;
+  m_cec_path = "/sys/devices/virtual/graphics/fb0/cec";
```
Comment (koying) [inline xbmc/powermanagement/android/AndroidPowerSyscall.cpp:34]: Could you put in comment for which device this cec sys path is, please.
Obviously, different devices will have something different.

Comment (elmerohueso) [inline xbmc/powermanagement/android/AndroidPowerSyscall.cpp:34]: Done

**Comment by koying** [comment]
Note quite able to test as I don't have a "truly" rooted device under hand but seems ok.
Please squash the commits into one.

**Comment by elmerohueso** [comment]
Something went wrong during rebase.  Closing request.  Will open new one when things are fixed.

**Pull request closed without merging**